evildmp/vanilla-sphinx-test
# Add syntax highlighting

## Done 

Add syntax highlighting styles following the same colour scheme as what we already have on Vanilla

## QA 

Go to `/repertoire` and scroll to the syntax highlighting part. See the code now has colour
